src/plugin.ts

```diff
@@ -181,7 +181,7 @@ export default plugin.withOptions<Partial<PluginOptions>>(
                                   }) as FontSizeOptions
 
                             return {
-                                '--font-size-px': fontSizeActual,
+                                '--font-size-px': String(fontSizeActual),
```

Comment:
Is there a benefit to using the `String` constructor vs a `.toString()` method?

Reply:
`String()` is nicer sometimes because internally it does the same thing as `toString()` but doesn't need a value to be defined to run successfully.

If the value is `undefined`, `toString()` will throw an error. `String()` will return `undefined`.

Does this matter in this case necessarily? I don't think so, so it's just a matter of preference.

Reply:
Sounds good to me 👍